core/templates/services/assets-backend-api.service.ts

```
@@ -134,13 +134,15 @@ export class AssetsBackendApiService {
     }));
     let imageUploadUrlTemplate = '/createhandler/imageupload/' +
     '<entity_type>/<entity_id>';
-    return this.http.post<{filename: string}>(
-      this.urlInterpolationService.interpolateUrl(
-        imageUploadUrlTemplate, {
-          entity_type: entityType,
-          entity_id: entityId
-        }
-      ), form);
+    let thumbnailFileUrl = this.urlInterpolationService.interpolateUrl(
+      imageUploadUrlTemplate, {
+        entity_type: entityType,
+        entity_id: entityId
+      });
+    if (thumbnailFileUrl === null) {
```

Why are we expecting `thumbnailFileUrl` to be null?

---

because `this.urlInterpolationService.getInterpolateUrl` can return a null value if arguments passed are not correct!

---

is that case we should raise an error, right?

---

@DubeySandeep done